packages/devtools_app/test/app_size/app_size_screen_test.dart

```diff
@@ -120,6 +128,34 @@ void main() {
       expect(splitter.initialFractions[0], equals(0.67));
       expect(splitter.initialFractions[1], equals(0.33));
     });
+
+    testWidgetsWithWindowSize('builds deferred content', windowSize,
+        (WidgetTester tester) async {
+      await pumpAppSizeScreen(
+        tester,
+        controller: appSizeController,
+      );
+
+      expect(find.byType(AppSizeBody), findsOneWidget);
+      expect(find.byType(TabBar), findsOneWidget);
+
+      expect(find.byKey(AppSizeScreen.analysisTabKey), findsOneWidget);
+      expect(find.byKey(AppSizeScreen.diffTabKey), findsOneWidget);
+
+      await loadDataAndPump(tester, data: deferredAppFile);
+
+      // Verify the state of the splitter.
+      final splitFinder = find.byType(Split);
+      expect(splitFinder, findsOneWidget);
+      final Split splitter = tester.widget(splitFinder);
+      expect(splitter.initialFractions[0], equals(0.67));
+      expect(splitter.initialFractions[1], equals(0.33));
+
+      // Verify the dropdown for selecting deferred units exists.
+      final appUnitDropdownFinder =
+          find.byKey(AppSizeScreen.appUnitDropdownKey);
```

Comment:
I would also verify that this dropdown doesn't show up when we are on the diff tab

Reply:
Done